### PR TITLE
chore: add docker-compose file for postgres & redis

### PR DESCRIPTION
**Pull request by lindskogen**
## How has this been tested?

- [ ] Unit tests
- [ ] Manually

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+services:
+  redis:
+    image: redis
+    ports:
+      - "6379:6379"
+  postgres:
+    image: postgres:13
```
Comment (Edholm) [inline docker-compose.yml:7]:
```suggestion
    image: postgres:14
```

That is the version I used when developing and also the version used in prod/non-prod.

It should work with 13 also but...

**Review by Edholm — APPROVED** [review]
Noice, maybe switch postgres version though

File: docker-compose.yml (hunk 1 above)
Comment (lindskogen) [reply docker-compose.yml:7]: We should change the README then :)
https://github.com/Filmstund/filmstund/blob/master/README.md?plain=1#L70-L78

Comment (Edholm) [reply docker-compose.yml:7]: :sweat_smile:

**Review by Edholm — APPROVED** [review]
:+1:

**Pull request merged**